number_trivia_app/lib/features/number_trivia/data/models/number_trivia_model.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:meta/meta.dart';
+
+import '../../domain/entities/number_trivia.dart';
+
+class NumberTriviaModel extends NumberTrivia {
+  NumberTriviaModel({
+    @required String text,
+    @required int number,
+  }) : super(text: text, number: number);
+
+  factory NumberTriviaModel.fromJson(Map<String, dynamic> json) {
+    return NumberTriviaModel(text: json['text'], number: (json['number'] as num).toInt());
```

Comment:
Should you add the named parameters on different lines?

Reply:
We discussed the topic offline